### PR TITLE
14899 used reactjs and ui-kit for report

### PR DESCRIPTION
**Pull request by saricicekibrahim**

### CODE REVIEW

File: make_profiler/report/src/components/report.jsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        this.handleSorting(accessor, sortOrder);
+    };
+
+    handleSorting = (sortField, sortOrder) => {
+        if (sortField) {
+            const sorted = [...this.state.filteredStatus].sort((a, b) => {
+                if (a[sortField] === null) return 1;
+                if (b[sortField] === null) return -1;
+                if (a[sortField] === null && b[sortField] === null) return 0;
+                if (a[sortField] === undefined || b[sortField] === undefined) return 0;
```
Comment (IlyaIzr) [inline make_profiler/report/src/components/report.jsx:62]: 3rd field will never happen

File: make_profiler/report/README.md [hunk 2]
```diff
@@ -11,4 +11,76 @@ index.html consumes and reports last status.
   
 Any process can be searched on UI.  
 Failed & idle tasks are reported with red & yellow colors.  
-All fields can be sorted.  
\ No newline at end of file
+All fields can be sorted.  
+
+
+# Getting Started with Create React App
```
Comment (Komzpa) [inline make_profiler/report/README.md:17]: this is a stub and stubs should not be

File: make_profiler/report/public/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (Komzpa) [inline make_profiler/report/public/index.html:10]: stub

File: make_profiler/report/src/App.test.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```
Comment (Komzpa) [inline make_profiler/report/src/App.test.js:4]: stub

**Review by Komzpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**